src/features/amUI/userRightsPage/DelegationModal/ResourceInfo.tsx

```diff
@@ -49,22 +49,50 @@ export const ResourceInfo = ({ resource, toParty, onDelegate }: ResourceInfoProp
   const [delegateRights] = useDelegateRightsMutation();
   const [rights, setRights] = useState<ChipRight[]>([]);
   const { openSnackbar } = useSnackbar();
+  const { data: reportee } = useGetReporteeQuery();
+
+  const [missingAccessMessage, setMissingAccessMessage] = useState<string | null>(null);
   const displayResourceAlert =
     error.isError ||
     resource?.delegable === false ||
     (rights.length > 0 && !rights.some((r) => r.delegable === true));
+
   const resourceRef: ResourceReference | null =
     resource !== undefined
       ? {
           resource: resource.authorizationReference,
         }
       : null;
 
+  const getMissingAccessMessage = (response: DelegationAccessResult[]) => {
+    const hasMissingRoleAccess = response.some((result) =>
+      result.details?.some((detail) => detail.code === 'MissingRoleAccess'),
```

Comment:
```suggestion
      result.details?.some((detail) => detail.code === 'MissingRoleAccess' || detail.code === 'MissingRightAccess'),
```
There are two cases that should result in the same message to the user